Project/Front-End/career-connect/src/pages/UserPages/ApplicantSignUp.jsx

```diff
@@ -0,0 +1,86 @@
+import React, { useState } from "react";
+import { useNavigate } from "react-router-dom";
+import { api } from "../../utilities";
+import { userContext } from "../../App";
+import { useContext } from "react";
+export default function ApplicantSignUpPage() {
+    const navigate = useNavigate();
+    const [userName,setUserName] = useState("");
+    const [password, setpassword ] = useState("");
+    const [name, setName ] = useState("");
+    const [ education, setEducation ] = useState("");
+    const [ school , setschool ] = useState("");
+    const [ field, setfield ] = useState("");
+    const { setUser } = useContext(userContext);
+    const accounttype = "applicant";
+    //  function to sign up user
+    const signUp = async(e) => {
+        //  prevent page reload
+        e.preventDefault();
+        //  pings backend api to create user
+        try{
+        let response = await api.post("users/sign-up/",{
+            email : userName,
+            password : password,
+            account_type : accounttype,
+            name : name,
+            education : education,
+            field : field,
+            school : school,
+            skills : "NONE",
+        });
+        let user = response.data.user;
+        let token = response.data.token;
+        setUser(user);
+        localStorage.setItem("token", token);
+        api.defaults.headers.common["Authorization"] = `Token ${token}`;
+        console.log("User created");
+        navigate("/ProfilePage");
+      }
+      catch{
+        window.location.reload(true);
```

Comment:
You might consider adding a more useful response to an erroneous signup. Reloading the page might not indicate to users what went wrong or what they're supposed to do next. A quick solution would be to add an alert message with `window.alert("msg...")`. The best solution would be to add an error pane on the screen with an appropriate message telling the user what occurred (this would be a stretch goal).